WhaleSpotting/Controllers/UserController.cs

```diff
@@ -40,4 +42,18 @@ public IActionResult Create([FromBody] UserRequest newUser)
         var url = Url.Action("GetById", new { userId = user.Id });
         return Created(url, new UserResponse(user));
     }
+
+    [HttpGet("ListUsers")]
+    public ActionResult<List<UserResponse>> ListOfUsers()
+    {
+        try
+        {
+            var usersList = _userService.ListAllUsers();
+            return Ok((usersList));
```

Comment:
Extra brackets here!
```suggestion
            return Ok(usersList);
```